module-bluetooth/Bluetooth/interface/profiles/A2DP/AVRCP.hpp

```diff
@@ -19,22 +19,14 @@ namespace bluetooth
 {
     class AVRCP
     {
-      public:
-        constexpr static const uint8_t subunitInfo[] = {0, 0, 0, 0, 1, 1, 1, 1, 2, 2, 2, 2, 3, 3, 3, 3,
-                                                        4, 4, 4, 4, 5, 5, 5, 5, 6, 6, 6, 6, 7, 7, 7, 7};
-        constexpr static uint32_t companyId          = 0x112233;
-        constexpr static uint8_t companiesNum        = 1;
-        constexpr static uint8_t companies[]         = {
-            0x00, 0x19, 0x58 // BT SIG registered CompanyID
-        };
+      private:
+        static std::uint16_t targetVolumeToPercent(std::uint16_t volume);
+        static std::uint16_t controllerVolumeToPercent(std::uint16_t volume);
 
-        constexpr static uint8_t eventsNum     = 6;
-        constexpr static uint8_t events[]      = {AVRCP_NOTIFICATION_EVENT_PLAYBACK_STATUS_CHANGED,
```

Comment:
are those events unnecessary now?

Reply:
These variables were unused after changes introduced by [this commit](https://github.com/mudita/MuditaOS/commit/9a6b8363351d722f037d4dbe5dd1f1cac25476ac), so I removed them as a part of cleanup. Three of these events are still in use, but the way they're registered has changed - see lines 45-48 of AVRCP.cpp.